### PR TITLE
View/search

### PR DESCRIPTION
**Pull request by titor-oopart**
was created the  search video frame class, main class run this frame implemented

### CODE REVIEW

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ */
+public class SearchVideoFrame extends JFrame {
+
+    private JPanel contentSearchVideoPane;
+
+    public SearchVideoFrame() {
+
+        initComponent();
+        initSetting();
+
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: Unnecessary empty line

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    /**
+     * This method init all component
+     */
+    public void initComponent() {
+
+        contentSearchVideoPane = new JPanel();
+
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: Unnecessary empty line

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+     * This method init all setting of the components
+     */
+    public void initSetting() {
+
+        setBounds(100, 100, 1000, 700);
+        contentSearchVideoPane.setBorder(new EmptyBorder(5, 5, 5, 5));
+        contentSearchVideoPane.setLayout(new BorderLayout(0, 0));
+        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        setContentPane(contentSearchVideoPane);
+
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: Unnecessary empty line

File: src/main/java/com/fundation/logic/Main.java [hunk 4]
```diff
@@ -9,14 +9,17 @@
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/Main.java:9]: Implement the copy write

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+import java.awt.BorderLayout;
+
+/**
+ * Implements the SearchVideoFrame class
+ *
+ * @author John Salazar Pinto
+ * @version 1.0
+ */
+public class SearchVideoFrame extends JFrame {
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: Remove this line.

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    private JPanel contentSearchVideoPane;
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: Remove this line.

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    public SearchVideoFrame() {
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: Remove this line.

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java (hunk 1 above)
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: Remove this line.

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+        SearchVideoPanel mainSearchPanel = new SearchVideoPanel();
+        mainSearchPanel.setVisible(true);
+        add(mainSearchPanel);
+
+        SearchVideoPanel initSearchPanel = new SearchVideoPanel();
+        contentSearchVideoPane.add(initSearchPanel);
+    }
+
+    /**
+     * This method init all setting of the components
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: Please explain what setting involve

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+            txtFldFrameRate, txtFldVideoFormat, txtFldResolution;
+    private JLabel lblSearch, lblPath, lblFileName, lblSize, lblCreationDate, lblModificationDate,
+            lblVideoSearchTittle, lblFrameRate, lblVideoFormat, lblResolution, lblAudio, lblAurioSearchTittle,
+            lblAudioChanel, lblAudioFileFormat;
+    private JComboBox comboSize, comboAudioFileFormat, comboAudioChanel;
+    private JCheckBox chxYes, chxNo;
+    private Integer positionOneX, positionTwoX, positionThreeX;//position x default
+
+    public SearchVideoPanel() {
+        initComponent();
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java:36]: Empty line required according on our code conventions

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 10]
```diff
@@ -0,0 +1,168 @@
...
+        add(comboAudioChanel);
+        comboAudioFileFormat = new JComboBox();
+        add(comboAudioFileFormat);
+
+        ////Init and adds check box
+        chxNo = new JCheckBox("No");
+        add(chxNo);
+        chxYes = new JCheckBox("Yes");
+        add(chxYes);
+
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: Unnecessary empty line

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 11]
```diff
@@ -0,0 +1,168 @@
...
+import javax.swing.JCheckBox;
+
+/**
+ * Implements the SearchVideoPanel class
+ *
+ * @author John Salazar Pinto
+ * @version 1.0
+ */
+public class SearchVideoPanel extends JPanel {
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: Remove this line

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 12]
```diff
@@ -0,0 +1,168 @@
...
+        lblAudio = new JLabel("Audio");
+        add(lblAudio);
+        //lblAudio search options
+        lblAurioSearchTittle = new JLabel("Specific audio search");
+        add(lblAurioSearchTittle);
+        lblAudioChanel = new JLabel("Audio chanel");
+        add(lblAudioChanel);
+        lblAudioFileFormat = new JLabel("Audio File Format");
+        add(lblAudioFileFormat);
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: Remove this line.

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 13]
```diff
@@ -0,0 +1,168 @@
...
+        txtFldModificationDate = new JTextField();
+        add(txtFldModificationDate);
+        //Init and adds Texts Fields Video search
+        txtFldFrameRate = new JTextField();
+        add(txtFldFrameRate);
+        txtFldVideoFormat = new JTextField();
+        add(txtFldVideoFormat);
+        txtFldResolution = new JTextField();
+        add(txtFldResolution);
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: Remove this line.

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 14]
```diff
@@ -0,0 +1,168 @@
...
+        //Init and adds combo box
+        comboSize = new JComboBox();
+        add(comboSize);
+        comboAudioChanel = new JComboBox();
+        add(comboAudioChanel);
+        comboAudioFileFormat = new JComboBox();
+        add(comboAudioFileFormat);
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: Remove this line.

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 15]
```diff
@@ -0,0 +1,168 @@
...
+        initSetting();
+    }
+
+    /**
+     * This method init all component
+     */
+    public void initComponent() {
+
+        setLayout(null);
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: Remove this line

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 16]
```diff
@@ -0,0 +1,168 @@
...
+    public SearchVideoPanel() {
+        initComponent();
+        initSetting();
+    }
+
+    /**
+     * This method init all component
+     */
+    public void initComponent() {
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: Remove this line.

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 17]
```diff
@@ -0,0 +1,168 @@
...
+        chxYes = new JCheckBox("Yes");
+        add(chxYes);
+
+    }
+
+    /**
+     * This method init all settings
+     */
+    public void initSetting() {
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: Remove this line.

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 18]
```diff
@@ -0,0 +1,168 @@
...
+        lblSize.setBounds(positionOneX, 120, 40, 20);
+        lblCreationDate.setBounds(positionOneX, 150, 160, 20);
+        lblModificationDate.setBounds(positionOneX, 180, 160, 20);
+        //Video search label positions
+        lblVideoSearchTittle.setBounds(positionOneX, 210, 100, 20);
+        lblVideoFormat.setBounds(positionOneX, 240, 100, 20);
+        lblResolution.setBounds(positionOneX, 270, 100, 20);
+        lblFrameRate.setBounds(positionOneX, 300, 100, 20);
+        lblAudio.setBounds(positionOneX, 330, 100, 20);
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: Remove this line.

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 19]
```diff
@@ -0,0 +1,168 @@
...
+        txtFldModificationDate.setBounds(positionTwoX, 180, 60, 20);
+        //Video search text fields positions
+        txtFldFrameRate.setBounds(positionTwoX, 300, 100, 20);
+        txtFldVideoFormat.setBounds(positionTwoX, 240, 100, 20);
+        txtFldResolution.setBounds(positionTwoX, 270, 100, 20);
+        //Video search text fields positions
+        lblAurioSearchTittle.setBounds(positionOneX, 350, 170, 20);
+        lblAudioChanel.setBounds(positionOneX, 380, 100, 20);
+        lblAudioFileFormat.setBounds(positionOneX, 410, 100, 20);
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: Remove this line.

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 20]
```diff
@@ -0,0 +1,168 @@
...
+        //text inside "TextFields"
+        txtFldPath.setText("...");
+        txtFldFileName.setText("...");
+        txtFldSize.setText("...");
+        txtFldCreationDate.setText("...");
+        txtFldModificationDate.setText("...");
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: Remove this line.

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 21]
```diff
@@ -0,0 +1,168 @@
...
+        //comboBoxes positions
+        comboSize.setBounds(positionThreeX, 120, 45, 20);
+        comboAudioChanel.setBounds(positionTwoX, 380, 60, 20);
+        comboAudioFileFormat.setBounds(positionTwoX, 410, 60, 20);
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: Remove this line.

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 22]
```diff
@@ -0,0 +1,168 @@
...
+import javax.swing.JTextField;
+import javax.swing.JCheckBox;
+
+/**
+ * Implements the SearchVideoPanel class
+ *
+ * @author John Salazar Pinto
+ * @version 1.0
+ */
+public class SearchVideoPanel extends JPanel {
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java:24]: This class does many things try to make have a structure and split by class.

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 23]
```diff
@@ -0,0 +1,168 @@
...
+    /**
+     * This method init all settings
+     */
+    public void initSetting() {
+
+        //positions to align labels textFields checkBoxes comboBoxes
+        positionOneX = 10;
+        positionTwoX = 180;
+        positionThreeX = 300;
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: Delete this line

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java (hunk 1 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java (hunk 2 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java (hunk 3 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: done

File: src/main/java/com/fundation/logic/Main.java (hunk 4 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/Main.java:9]: done

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java (hunk 5 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java (hunk 6 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java (hunk 7 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java (hunk 1 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java (hunk 8 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java (hunk 9 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoPanel.java:36]: done

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java (hunk 10 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: updated

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java (hunk 11 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: updated

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java (hunk 12 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java (hunk 13 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java (hunk 14 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java (hunk 15 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java (hunk 17 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java (hunk 18 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java (hunk 19 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java (hunk 20 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java (hunk 21 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java (hunk 22 above)
Comment (johnpiterzon) [reply src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java (hunk 23 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoPanel.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java (hunk 2 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java (hunk 1 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: done

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java (hunk 3 above)
Comment (titor-oopart) [reply src/main/java/com/fundation/logic/view/SearchVideoFrame.java]: done

**Pull request closed without merging**